hack/start_db.sh

```diff
@@ -8,4 +8,11 @@ mkdir -p /tmp/postgres/data
 mkdir -p /tmp/postgres/sockets
 
 initdb -D /tmp/postgres/data -U postgres
+
+# We store the data files in the `/tmp` directory assuming that it will be a `tmpfs` and therefore
+# very fast. But in some environments it may not be, as it may be mapped to a real persistent file
+# system. Disabling `fsync` speeds things up in those environment, at the cost of risking data loss,
```

Comment:
Maybe add a tiny bit more detail on the purpose of fysnc to this comment.

The context that this is ensuring a file is fully written even in the case of a node crash/power failure/reboot is relevant here as _we don't expect or need our data to survive a reboot_. This is the real reason we don't care about "data loss"

Reply:
The meaning of the `fsync` parameter is described in the documentation of PostgreSQ. For example, for version 12 it is here: https://www.postgresql.org/docs/12/runtime-config-wal.html . I was trying to explain that we don't care about data loss because this is used only in unit tests, and crash, power failure or reboot is not a concern in unit tests. Anyhow, this is in this pull request by accicent, not really needed. I will remove it in the next version to avoid noiise.

Reply:
Sure. I think my point was just to mention what you say about unit tests, the original text doesn't fully explain this and leaves some room for interpretation.